File: SuperiorPackGroup/Code Classes/WeeklyProductionList.vb
Public Class WeeklyProductionList
    Inherits List(Of WeeklyProduction)
    Public Sub New()
        
    End Sub
    Public Sub New(ByVal capacity As Integer)
        MyBase.New(capacity)
        
    End Sub
    Public Sub New(ByVal collection As IEnumerable(Of WeeklyProduction))
        MyBase.New(collection)
        
    End Sub

End Class

File: SuperiorPackGroup/Code Classes/WeeklyProductionList.cs
//INSTANT C# NOTE: Formerly VB project-level imports:
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Drawing;
using System.Diagnostics;
using System.Windows.Forms;

namespace SuperiorPackGroup
{
	public class WeeklyProductionList : List<WeeklyProduction>
	{
		public WeeklyProductionList()
		{

		}
		public WeeklyProductionList(int capacity) : base(capacity)
		{

		}
		public WeeklyProductionList(IEnumerable<WeeklyProduction> collection) : base(collection)
		{

		}

	}

}